.travis.yml

```diff
@@ -28,7 +28,7 @@ install:
  - curl -sSf https://sh.rustup.rs | sh -s -- -y
  - export PATH=$HOME/.cargo/bin:$PATH
  - rustc --version
- - (cd js; yarn)
+ - (cd third_party; yarn)
  - (cd third_party; gclient sync -j2 --no-history)
```

Comment:
You should be able to replace these lines with `./tools/build_third_party.py` ?

Reply:
If dropping `-j2` doesn't cause a problem, then that should work. I'll try that on this PR.

Reply:
Yes - that should be fine. (I think gclient is `-j1` by default and I was just testing to see if `-j2` made things faster - it doesn't - but it got left in)